pkg/controllers/vdb/onlineupgrade_reconciler.go

```diff
@@ -1129,14 +1124,6 @@ func (r *OnlineUpgradeReconciler) promoteSandboxToMainCluster(ctx context.Contex
 	var res ctrl.Result
 	res, err = actor.Reconcile(ctx, &ctrl.Request{})
 	if verrors.IsReconcileAborted(res, err) {
```

Comment:
We should only increase attempt number when there is an error. If a pod is not up and res is ctrl.Result{Requeue: true}, we don't want to increase the attempt number.